app/views/disciplines/index.html.erb

```diff
@@ -23,13 +23,16 @@
         <td><%= discipline.cargaHoraria %></td>
         <td><%= discipline.periodo %></td>
         <td><%= link_to 'Show', discipline %></td>
-        <td><%= link_to 'Edit', edit_discipline_path(discipline) %></td>
-        <td><%= link_to 'Destroy', discipline, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <% if current_user.person.perfil == Person.perfils.key(0) %>
```

Comment:
aqui eu acho que poderia ser assim... como é um enum vc pode verificar dessa maneira:
`<% if current_user.person.aluno %>`

Reply:
@talithamedeiros  eu aceitei mesmo assim. Despois testa ai no seu e se for o caso corrigimos depois.. Dessa forma fica mais intuitivo.

Reply:
Ele não consegue acessar, ele considera que aluno é um método, aparece a mensagem de erro:
undefined method `aluno'.
Tem que ser dessa forma mesmo.